src/procedures/proc_vector_create_index.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright FalkorDB Ltd. 2023 - present
+ * Licensed under the Server Side Public License v1 (SSPLv1).
+ */
+
+#include "../value.h"
+#include "../errors.h"
+#include "../query_ctx.h"
+#include "../index/indexer.h"
+#include "../datatypes/map.h"
+#include "../graph/graphcontext.h"
+#include "proc_vector_create_index.h"
+
+// parse procedure arguments
+static bool _parseArgs
```

Comment:
can we merge this with proc_fulltext_create_index just pass the index type? maybe also move this to a command GRAPH.INDEX CREATE and get read of the parsing of query

Reply:
full-text index and vector index have a different set of parameters, I don't think we should merge the two, what I would like is a consistent API for managing indices:
1. CREATE
2. QUERY
3. DELETE

Today there's a distinction between all 3 types of indices (EXACT-MATCH, FULL-TEXT and VECTOR).

Neo introduced:
`db.index.vector.createNodeIndex()` and `db.index.vector.queryNodes()`

I'll get myself familiarized with Neo's API